README.md

```diff
@@ -129,13 +129,26 @@ The default configuration assumes `mainnet` with only `btc` chain supported, and
 You can change the location of the configuration file with the `--conf=pathToConf` command line argument.
 
 ### Command line parameters
+Please note that NBXplorer uses cookie authentication by default. If you run your Bitcoin/Litecoin/Dash nodes using their daemon (like `bitcoind`, `litecoind` or `dashd`), they generate a new cookie every time you start them, and that should work without any extra configuration. 
+If you run the node(s) using the GUI versions, like Bitcoin\Litecoin\Dash Core Qt with the `-server` parameter while you set the rpcusername and rpcpassword in their `.conf` files, you must set those values for every crypto you are planning to support. 
+See samples below.
 
-#### From Source (.NET Core SDK required)
-The same settings as above, e.g.
-```dotnet run NBXplorer.dll --port=20300 --network=mainnet --nodeendpoint=127.0.0.1:32939```
+#### Run from source (.NET Core SDK required)
+```dotnet run NBXplorer.dll <parameters>```
+
+#### Run using built DLL (.NET Core Runtime required)
+```dotnet NBXplorer.dll <parameters>```
```

Comment:
`run.sh` scripts instead.

Reply:
I think this is useful as well. We should probably replace the `NBXplorer.dll` with `-p .\NBXplorer\NBXplorer.csproj`, but otherwise it's helpful.

Reply:
```dotnet run --no-launch-profile -p .\NBXplorer\NBXplorer.csproj  -- <parameters>```